src/features/Classes/ClassesPage/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import Container from '@edx/paragon/dist/Container';
+import { Pagination } from '@edx/paragon';
```

Comment:
```suggestion
import { Pagination, Container } from '@edx/paragon';
```
Please move Container import with Pagination